packages/get-platform/src/getPlatform.ts

```diff
@@ -52,6 +53,14 @@ export async function getos(): Promise<GetOSResult> {
     }
   }
 
+  if (platform === 'android') {
+    return {
+      platform: 'linux',
+      libssl: await getOpenSSLVersion(platform),
+      arch,
```

Comment:
I would expect (note: I did not check) that what uses these results, requires `distro` to be set when `platform == linux`. Agree?